src/index.ts

```diff
@@ -304,6 +311,7 @@ export interface TsConfigOptions
     | 'cwd'
     | 'projectSearchDir'
     | 'experimentalEsmLoader'
+    | 'experimentalReplAwait'
```

Comment:
I think we can allow users to specify this flag via tsconfig.  Do you foresee any problems there?

Reply:
At first hand no, this only affects REPL usage so I don't see any issues, should it also attempt to read from env?